metron-deployment/roles/mysql_server/tasks/geoip.yml

```diff
@@ -15,12 +15,21 @@
 #  limitations under the License.
 #
 ---
-- name: Download GeoIP databases
+- name: Download GeoIP databases from specified URL
   unarchive:
-    src:  http://geolite.maxmind.com/download/geoip/database/GeoLiteCity_CSV/GeoLiteCity-latest.tar.xz
+    src: "{{ geodata_file }}"
     dest: /tmp/geoip
     copy: no
     creates: /tmp/geoip/*/GeoLiteCity-Blocks.csv
+  when: '"{{ geodata_file }}".find("http://") != -1'
```

Comment:
A better way to do this check might be the following.  This approach is a little simpler and doesn't preclude the use of FTP or HTTPS.

```
when: geodata_file is defined
```